test/extended/registry/signature.go

```diff
@@ -0,0 +1,120 @@
+package registry
```

Comment:
yeah, i don't think either of these packages is a great location for this test, but I would like to have it in registry as the verification fetching manifest from registry and I want to add additional test in (near) future for the signature endpoint in registry (don't want to have the signature test splitted in two packages)